laze-project.yml

```
@@ -1413,6 +1421,18 @@ builders:
   - name: st-nucleo-f401re
     parent: stm32f401retx
 
+  - name: stm32f401cd
```

```suggestion
  - name: stm32f401cdux
```

These laze context names should match the values passed as `--cfg` just below; we'll later (soon?) rework this following #958 to shorten these, but for now we do need that redundancy.